pages/blog/[id].js

```diff
@@ -453,6 +436,42 @@ export default class Post extends React.Component {
   }
 }
 
+export async function getStaticPaths() {
+  // Call an external API endpoint to get posts
```

Comment:
Not sure you need comments like this.
The code should be self-documented. It'd be great for comments to explain "why?" not "what?" or "how?"